__stack.yaml

```diff
@@ -0,0 +1,9 @@
+# This default stack file is a copy of stack-ghc8.6.5.yaml
```

Comment:
Can you clean up your stack files so that they are just like they were? I don't think they need any changes

Reply:
Yes, will edit it now. Thanks !